src/faction_camp.cpp

```diff
@@ -717,6 +718,15 @@ void basecamp::get_available_missions( mission_data &mission_key, bool by_radio
                                       "Items that do not have a loot destination zone will be sorted "
                                       "using the normal rules for automatic zone sorting." ) );
             mission_key.add( "Reset Sort Points", _( "Reset Sort Points" ), entry );
+            validate_assignees();
+            std::vector<npc_ptr> npc_list = get_npcs_assigned();
+            entry = string_format( _( "Notes:\n"
+                                      "Assign repeating job duties to NPCs stationed here. "
```

Comment:
Missing a linebreak?
```suggestion
                                      "Assign repeating job duties to NPCs stationed here.\n"
```